docs/interactionmanager.md

```diff
@@ -37,6 +37,157 @@ By default, queued tasks are executed together in a loop in one `setImmediate` b
 
 ---
 
+## Example
+
+### Basic
+
+```SnackPlayer name=InteractionManager%20Function%20Component%20Basic%20Example&supportedPlatforms=ios,android
+import * as React from 'react';
```

Comment:
This is a bad example, please use `import React, { ... }` where `...` should be replaced by used hooks.